README.md

```diff
@@ -1,57 +1,117 @@
 
+2. Install the package:
+   ```bash
+   pip install -e .
+   ```
+3. To update the project codebase (for existing project)
+    ```bash
+   git pull origin main
+   ```
 
-**For directly installing from pip**
+4. To update dependencies used by the project (for existing project):
+   ```bash
+   pip install -e . --upgrade
+   ```
 
-You can install through to execute directly
+### From PyPI
 
-`pip install git+https://github.com/PolicyEngine/policyengine-taxsim.git`
+```bash
+pip install git+https://github.com/PolicyEngine/policyengine-taxsim.git
+```
 
-### Example ##
-input file:
+## Usage
 
-<img width="641" alt="Screenshot 2024-07-10 at 6 29 17 PM" src="https://github.com/sgerson2/policyengine-taxsim/assets/113052102/db0ee3e4-9a54-42e7-a4fc-e46f07ab83f8">
+Run the simulation by providing your input CSV file:
 
+```bash
+python policyengine_taxsim/cli.py your_input_file.csv
+```
 
-## List of working input variables ##
+The output will be generated as `output.csv` in the same directory.
 
-### Demographics: ###
-1. taxsimid 
-2. year
-3. state
-4. mstat (only 1 (single) and 2 (joint) filing options work)
-5. page (age of primary taxpayer)
-6. sage (age of spouse)
-7. depx (number of dependents)
-8. age1 (age of first dependent)
-9. age2 (age of second dependent)
-10. age3 (age of third dependent)
+## Input Variables
 
-### Income: ###
-1. pwages (wage of primary taxpayer)
-2. swages (wage of spouse)
-3. psemp (self-employment income of primary taxpayer)
-4. ssemp (self-employment income of spouse)
-5. dividends (dividend income)
-6. intrec (taxable interest received)
-7. stcg (short-term capital gains)
-8. ltcg (long-term capital gains)
-9. pui (primary taxpayer unemployment compensation received)
-10. sui (spouse unemployment compensation received)
-11. proptax (real estate taxes paid)
+The emulator accepts CSV files with the following variables:
+
+### Demographics
+
+| Variable  | Description                     | Notes                                    |
+|-----------|--------------------------------|------------------------------------------|
+| taxsimid  | Unique identifier              |                                          |
+| year      | Tax year                       |                                          |
+| state     | State code                     |                                          |
+| mstat     | Marital status                 | Only supports: 1 (single), 2 (joint)     |
+| page      | Primary taxpayer age           |                                          |
+| sage      | Spouse age                     |                                          |
+| depx      | Number of dependents           |                                          |
+| age1      | First dependent's age          |                                          |
+| age2      | Second dependent's age         |                                          |
+| ageN      | Nth dependent's age            |                                          |
```

Comment:
this works for any N? taxsim works for up to 8, if we do the same could you add as a note?